packages/search-in-workspace/src/browser/search-in-workspace-frontend-contribution.ts

```diff
@@ -343,6 +369,14 @@ export class SearchInWorkspaceFrontendContribution extends AbstractViewContribut
             keybinding: 'shift+alt+f',
             when: 'explorerResourceIsFolder'
         });
+        keybindings.registerKeybinding({
+            command: SearchInWorkspaceCommands.FOCUS_NEXT_RESULT.id,
+            keybinding: 'f4',
+        });
+        keybindings.registerKeybinding({
+            command: SearchInWorkspaceCommands.FOCUS_PREV_RESULT.id,
+            keybinding: 'shift+f4',
+        });
```

Comment:
We should add the `when` context to prevent the keybinding from applying in all contexts. Before, the keybinding would still apply, but would do nothing due to the `isEnabled` of the command. This way commands can still use <kbd>f4</kbd> and <kbd>shift</kbd>+<kbd>f4</kbd> if they have a when clause which has precedence:

```suggestion
        keybindings.registerKeybinding({
            command: SearchInWorkspaceCommands.FOCUS_NEXT_RESULT.id,
            keybinding: 'f4',
            when: 'hasSearchResult'
        });
        keybindings.registerKeybinding({
            command: SearchInWorkspaceCommands.FOCUS_PREV_RESULT.id,
            keybinding: 'shift+f4',
            when: 'hasSearchResult'
        });
```